From: Benjamin Kaduk <kaduk@mit.edu>
Date: Tue, 24 Mar 2020 15:33:59 -0700
Subject: Temporarily disable flaky test

The volser test consistently fails on the armhf buildd but succeeds
on the porterbox.  The nature of the test in question suggests that
the issue in the test triggers only on a machine with a specific
network configuration using specific addresses in 10.0.0.0/8, which
should help track down the issue causing the test to fail on the buildd.

Change-Id: Ie6403deb3eea62d3ab01ac45be502ae2fd94080a
---
 tests/TESTS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/TESTS b/tests/TESTS
index e59014e..07277e1 100644
--- a/tests/TESTS
+++ b/tests/TESTS
@@ -18,6 +18,5 @@ ptserver/pts-man
 rx/event
 rx/perf
 volser/vos-man
-volser/vos
 bucoord/backup-man
 kauth/kas-man
